### PR TITLE
Update Readme with badge for Scala support, fix other images

### PR DESCRIPTION
**Pull request by rtyley**
This changes the Scaladex badge in the README to show which versions of Cats support what versions of Scala. I've chosen the variant which also indicates Scala.js support, as Cats is being cross-published for exactly the same Scala versions for the JVM and Scala.js. At the time of writing, the badge says:

```
cats-core-2.6.1 (Scala 3.x, 2.13, 2.12 - Js 1.x), 2.0.0 (Scala 2.11 - Js 0.6), 1.2.0 (Scala 2.10 - Js 0.6)
```
[cats-core Scala version support](https://index.scala-lang.org/typelevel/cats/cats-core)
More details on the badge format here: https://github.com/scalacenter/scaladex/pull/660

As well as this, I've fixed up a few broken images on the Cats Readme - see inline comments.

cc @larsrh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-### ![#f03c15](https://placehold.it/15/f03c15/000000?text=+) Community Announcements ![#f03c15](https://placehold.it/15/f03c15/000000?text=+)
```
Comment (rtyley) [inline README.md:2]: Images around the 'Community Announcements' title, introduced with https://github.com/typelevel/cats/pull/2683, and served by https://placehold.it/ are unfortunately broken:
image

File: README.md [hunk 2]
```diff
@@ -10,12 +10,10 @@
 
 ## Cats
 
-![cats image](http://plastic-idolatry.com/erik/cats2.png)
+![Cats graphic](https://user-images.githubusercontent.com/52038/121751434-05a6ec80-cb06-11eb-9b7e-d9f126a0a4e4.png)
```
Comment (rtyley) [inline README.md:13]: The Cats graphic at http://plastic-idolatry.com/erik/cats2.png used HTTP, so was considered insecure by modern browsers on the HTTPS web page https://typelevel.org/cats/ - leading to another broken image:
image

Comment (larsrh) [reply README.md:13]: Should we put the Cats image into the repository? Or where does the `githubusercontent.com` address come from?

Comment (rtyley) [reply README.md:13]: `user-images.githubusercontent.com` is [the domain that GitHub uses](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/about-anonymized-urls) to serve the images that we [copy & paste into GitHub issues/PRs](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files):

image

[A user can't change or delete an image](https://github.community/t/does-user-images-githubusercontent-expires/129292) once it's uploaded this way, and I believe you can think of them as being nearly as permanent as the existence of GitHub. I'm so lazy, I often copy-and-paste an image into an empty GitHub issue just to have somewhere public to serve the image from.

It's true that it's a rather opaque reference! If we were to put the image in the repo, whereabouts would it belong? All the current images in the project are in [`/docs/src/main/resources/microsite/img`](https://github.com/typelevel/cats/tree/main/docs/src/main/resources/microsite/img) - which gives urls that work for the microsite, eg https://typelevel.org/cats/img/sidebar_brand.png - should the ReadMe markdown refer to that address? An alternative would be the 'raw' GitHub url: https://raw.githubusercontent.com/typelevel/cats/main/docs/src/main/resources/microsite/img/sidebar_brand.png

Comment (larsrh) [reply README.md:13]: I think the raw URL is a good compromise, it should render correctly in both situations (website and README).

Comment (rtyley) [reply README.md]: Done with 26ba0d9ecc86fe1751d4205d94533b1d421e5a24 - the image will appear broken [in markdown preview](https://github.com/typelevel/cats/blob/26ba0d9ecc86fe1751d4205d94533b1d421e5a24/README.md#cats) until the PR is merged!

image

**Review by larsrh — APPROVED** [review]

**Comment by larsrh** [comment]
Looks good, thanks!

**Pull request merged**